src/webcomponents/api/rest-endpoint.js

```diff
@@ -525,11 +525,18 @@ export default class RestEndpoint extends LitElement {
 
         // Add QUERY params
         this.endpoint.parameters
-            .filter(parameter => parameter.param === "query" && this.data.param[parameter.name])
+            .filter(parameter => (parameter.param === "query" && parameter.name !== "study") && this.data.param[parameter.name])
             .forEach(parameter => {
                 url += `&${parameter.name}=${this.data.param[parameter.name]}`;
             });
 
+        // check if QUERY is required and it is not exist in the data
+        this.endpoint.parameters
+            .filter(parameter => (parameter.name === "query" && parameter.required) && this.data.param[parameter.name] === undefined)
```

Comment:
1. Are you sure that the condition `parameter.name === "query"` is correct?
2. Please use `typeof this.data.param[parameter.name] === "undefined"` instead.